packages/grafbase-sdk/README.md

```diff
@@ -462,7 +471,7 @@ const derp = auth.JWKS({
 Everywhere where one can define authentication rules, it happens through a lambda with a rules builder.
 
 ```typescript
-(rules) => {
+;(rules) => {
```

Comment:
typo?

Reply:
Not really, it's just formatted correctly with Prettier since the example shown is an IIFE.

Reply:
@fbjork I've updated it to be shown differently so it doesn't add the pesky `;`